webapp/channels/src/components/self_hosted_purchases/self_hosted_expansion_modal/self_hosted_expansion_modal.scss

```diff
@@ -0,0 +1,185 @@
+.SelfHostedExpansionModal {
+    height: 100%;
+
+    .form-view {
+        display: flex;
+        width: 100%;
+        height: 100%;
+        flex-direction: row;
+        flex-grow: 1;
+        flex-wrap: wrap;
+        align-content: top;
+        justify-content: center;
+        padding: 77px 107px;
+        color: var(--center-channel-color);
+        font-family: "Open Sans";
+        font-size: 16px;
+        font-weight: 600;
+        overflow-x: hidden;
+
+        .title {
+            font-size: 22px;
+            font-weight: 600;
+        }
+
+        .form {
+            padding: 0 96px;
+            margin: 0 auto;
+
+            .form-row {
+                display: flex;
+                width: 100%;
+                margin-bottom: 24px;
+            }
+
+            .form-row-third-1 {
+                width: 66%;
+                max-width: 288px;
+                margin-right: 16px;
+
+                .DropdownInput {
+                    z-index: 99999;
```

Comment:
You should be able to drop this z-index rule and the one in the dropdown input below: I think between the time you re-used this code and now, we had a change to the way this is done to make it work with the up to 4 dropdown inputs in the modal: https://github.com/mattermost/mattermost-server/pull/22626/files#diff-2cc2eda2e9fc02a8a66142400304bf6bd925ea8fdc93336263b820ceb196560bR1